api/src/main/java/org/apache/iceberg/rest/CreateNamespaceResponse.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.rest;
+
+import java.util.Map;
+import org.apache.iceberg.catalog.Namespace;
+import org.apache.iceberg.relocated.com.google.common.base.Preconditions;
+
+/**
+ * Represents a REST response to a create a namespace / database request.
+ *
+ * The properties returned will include all the user provided properties from the
+ * request, as well as any server-side added properties.
+ *
+ * Example server-side added properties could include things such as "created-at"
+ * or "owner".
+ *
+ * Presently, the JSON looks as follows:
+ *  { "namespace": "ns1.ns2.ns3", "properties": { "owner": "hank", "created-at": "1425744000000" } }
+ *
+ * Eventually, the idea is to wrap responses in IcebergHttpResponse so that it has a more standardized
+ * structure, including the possibility of richer metadata on errors, such as tracing telemetry or follw
+ * up user instruction.
+ *
+ * {
```

Comment:
I think you need <pre> HTML tag for these code blocks to show properly in javadoc.